kube_apiserver_metrics/README.md

```diff
@@ -68,3 +149,6 @@ Need help? Contact [Datadog support][11].
 [9]: https://docs.datadoghq.com/agent/faq/agent-commands/#agent-status-and-information
 [10]: https://github.com/DataDog/integrations-core/blob/master/kube_apiserver_metrics/metadata.csv
 [11]: https://docs.datadoghq.com/help/
+[12]: https://docs.datadoghq.com/containers/kubernetes/integrations/?tab=annotations
+[13]: https://docs.datadoghq.com/containers/cluster_agent/clusterchecks/?tab=helm#configuration-from-configuration-files
+[14]: https://containers/cluster_agent/clusterchecks/?tab=datadogoperator#setting-up-check-configurations
```

Comment:
Does this link [14] resolve correctly with that `https://`? I thought it had to be either the full URL like `http://docs.datadoghq.com/containers/cluster_agent/clusterchecks/?tab=datadogoperator#setting-up-check-configurations` or just the end path like `/containers/cluster_agent/clusterchecks/?tab=datadogoperator#setting-up-check-configurations`.

Reply:
oops yes good catch! I'll fix that.